contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -236,7 +235,7 @@ InstallExtendedHooks(void)
 	logicalrep_modify_slot_hook = logicalrep_modify_slot;
 
 	prev_is_tsql_rowversion_or_timestamp_datatype_hook = is_tsql_rowversion_or_timestamp_datatype_hook;
-	is_tsql_rowversion_or_timestamp_datatype_hook = is_tsql_rowversion_or_timestamp_datatype;
+	is_tsql_rowversion_or_timestamp_datatype_hook = common_utility_plugin_ptr->is_tsql_rowversion_or_timestamp_datatype;
```

Comment:
we should have init'd this hook in common extension side.

Reply:
We will not get anything extra even if we do that since the specific plugin (`is_tsql_rowversion_or_timestamp_datatype`) is anyway needed in tsql extensions in other places as well so it's not like we are reducing one function from the structure. But I will address this separately.